web/modules/custom/usergreeting/config/install/usergreeting.settings.yml

```diff
@@ -0,0 +1,4 @@
+usergreeting_defaults:
```

Comment:
The `usergreeting_defaults` key is unnecessary. Drupal will ignore it when dealing with the configuration.